Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Graph.cpp

```diff
@@ -316,7 +318,12 @@ namespace ScriptCanvas
 
             if (validationResults.HasResults())
             {
-                AZ_Error("ScriptCanvas", result.IsModelValid(), "Script Canvas parsing failed");
+                // reduce the noise of reporting against empty graphs
+                if (!(validationResults.ErrorCount() == 1
```

Comment:
if we don't log error for empty graph, do we handle empty graph in other ways? (like some notifications to tell user that graph is empty) if not, then have an EmptyGraph specific validation result doesn't make sense. Or we should log it in debug build?

Reply:
We track the error, still, and it is used in the builder to NOT build an empty asset, but we will update the reporting of it.